chain/network/src/peer_manager/peer_manager_actor.rs

```diff
@@ -119,6 +119,8 @@ const IMPORTANT_MESSAGE_RESENT_COUNT: usize = 3;
 // network issue.
 const UNRELIABLE_PEER_HORIZON: u64 = 60;
 
+const SKIP_TOMBSTONES_AFTER_STARTUP_TIME: time::Duration = time::Duration::seconds(120);
```

Comment:
you might actually want to put this value into the JSON config, rather than just a boolean flag.
In particular, putting a 0 here would behave the same way as disabling skipping, right?